index.html

```diff
@@ -25,9 +25,9 @@
               <li class="close-nav"> <img src="images/close.png" alt="Close Menu"> </li>
           </ul>
           <ul class="mobile-list">
-              <li class="nav-items"><a href="#recent-work" class="desktop-menu"> Portfolio </a></li>
-              <li class="nav-items"><a href="#aboutme" class="desktop-menu"> About </a></li>
-              <li class="nav-items"><a href="#contactme" class="desktop-menu"> Contact </a></li>
+              <li class="nav-items"><a href="#recent-work" class="mobile-menu"> Portfolio </a></li>
+              <li class="nav-items"><a href="#aboutme" class="mobile-menu"> About </a></li>
+              <li class="nav-items"><a href="#contactme" class="mobile-menu"> Contact </a></li>
           </ul>
```

Comment:
### OPTIONAL 
- Well done Team 👏🏿 

- [ ] Please make the background not scrollable  when the mobile nav popup is open currently I see it is, please check the video below 👍🏿 
- [ ] Since the background  of the popup is white the background page is not visible, but  adding this will make your project better 👍🏿 

https://user-images.githubusercontent.com/69638013/162947999-4482bda5-bc8e-46ea-ac46-5a9576cae773.mp4